os/alios/yallocbufferhandler.h

```diff
@@ -49,6 +49,9 @@ class YallocBufferHandler : public NativeBufferHandler {
   int32_t UnMap(HWCNativeHandle handle, void *map_data) const override;
 
   uint32_t GetFd() const override;
+  bool GetInterlace(HWCNativeHandle handle) const override {
```

Comment:
Is "handle" included for future use?  It is not being referenced here.

Reply:
It is a   virtual API . handle should be used if some implementation. See Android implementation